### ISSUE TITLE
Suggestion, sales.

### ISSUE THREAD

**Issue by raycohaex** [Facepunch/RustWorkshop#131]
Currently skin prices ascend every week, but could it be a experiment to make them descend instead?
This is still a broad concept so it would need more polishing I supose, which I will leave open for the rust workshop creators.

And yes, this will benefit the artists more than the rust-COMMUNITY-market.

**Comment by kfhammond** [comment]
I always though it might make sense for prices in the rust store to be at premium during the duration of time that skins can't yet be traded in the market. After that they could drop in price quickly.

**Comment by raycohaex** [comment]
Most items that are random drop sell for 20$, this means its more beneficial to grab it from the item store in theory. There is no point in selling a 6$ skin for 2$ either. Creators get more chance of getting an earning from their skin before it gets removed. After that it's given tot he community since it can be purchaged for 1$ last day and probaly stays around 1$ in the market too

**Comment by MDemonWorkshop** [comment]
start with high price, then descend. experiment please

**Comment by MDemonWorkshop** [comment]
I totally agree that when items are sold for $1 from the first day of sales, they bring little profit to both creators and artists, but don't you think that if you begin an experiment with starting off at high prices and then gradually descending them, nobody's going to buy items at the beginning because everyone will just be waiting for the last sales day when they could buy this item much, much cheaper? I mean, if you start with high prices, there will be no demand at the beginning of sales.

**Comment by raycohaex** [comment]
Then whats the point of starting with 1$, everyone will buy it the first day too. The chance people buy skins in what we concider now as "Week 2 and week3" will be higher when you start with 10$, people will buy the skin as soon as they concider the price as worthy.

**Comment by Kamiling** [comment]
Yeah, that can be nice.

**Comment by StabbyMcStabface** [comment]
I think people will just wait until it's cheapest and buy it (as many currently do when it hits "the market"). I  think that maybe keeping the price "fixed" and as a store exclusive for 4-5 weeks (no market release until it's retired) might be worth a go. People seem to like the reasonable pricing, so maybe not deviate from that too much (maybe a fixed price of $2.49?). I think the sales would still be there (and not lost to the market).  

That being said, I am noticing an uptick in sales of older items with the current system though, so it could be the economy is starting to normalize and supply/demand is kicking in.

**Comment by RavvickHipnot** [comment]
It strikes me that it would be nice if we made as much money as possible from our work before the profits go to people trading skins on the market. I mean, we did the work, right?

I've only had one accepted so far, and its initial sales were great. In the second week they've fallen off a cliff.
